srcpkgs/cqrlog/INSTALL.msg

```diff
@@ -0,0 +1,10 @@
+Depending on which database server you (want to) use, you must install
+the following packages:
+
+For mysql:
+
+	* mysql libmysqlclient-devel
+
+For mariadb:
+
+	* mariadb libmariadbclient-devel
```

Comment:
`-devel` required?

Reply:
cqrlog looks for the `libmysqlclient.so` symlink, which is (obviously) only in the `-devel` package.
Is there perhaps a better way to handle this?